project/Win32BuildSetup/BuildSetup.bat

```diff
@@ -157,6 +161,41 @@ set WORKSPACE=%CD%\..\..
   ECHO ------------------------------------------------------------
   GOTO MAKE_BUILD_EXE
 
+
+:COMPILE_CMAKE_EXE
+  ECHO Wait while preparing the build.
+  ECHO ------------------------------------------------------------
+  ECHO Compiling %APP_NAME% branch %BRANCH%...
+
+  IF %buildmode%==clean (
+    RMDIR /S /Q %WORKSPACE%\kodi-build
+  )
+  MKDIR %WORKSPACE%\kodi-build
+  PUSHD %WORKSPACE%\kodi-build
+
+  call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86
```

Comment:
don't think this is needed, cmake takes care of finding the build tools itself. If it must be here, should probably use %VS140COMNTOOLS%\..\..\VC\vcvarsall.bat x86